subprojects/core/src/main/java/org/gradle/api/internal/catalog/parser/TomlCatalogFileParser.java

```diff
@@ -254,12 +254,21 @@ private static void parseLibrary(String alias, TomlTable librariesTable, Version
         Object gav = librariesTable.get(alias);
         if (gav instanceof String) {
             List<String> splitted = SPLITTER.splitToList((String) gav);
-            if (splitted.size() == 3) {
+            int splittedSize = splitted.size();
+            if (splittedSize >= 2 && splittedSize <= 3) {
```

Comment:
```suggestion
            if (splittedSize == 2 && splittedSize == 3) {
```

Reply:
You mean that with a `||`, right? 😄

Reply:
yes ! Thank you
Way simpler to read I thought

Reply:
👍 change is pushed